serverless/cmd/integrate/main.go

```diff
@@ -58,11 +59,10 @@ func main() {
 	// Read log private key from file or environment variable
 	var privKey string
 	if len(*privKeyFile) > 0 {
-		k, err := ioutil.ReadFile(*privKeyFile)
+		privKey, err = getKeyFile(*privKeyFile)
 		if err != nil {
-			glog.Exitf("failed to read private_key file: %q", err)
+			glog.Exitf("unable to get public key: %q", err)
```

Comment:
error message needs updating to reflect that it's the priv key

Reply:
Done.